### DISCUSSION TITLE
Close Steam to tray

### DISCUSSION THREAD

**Discussion by metov** [i3/i3#5361] [Q&A]
I have a Win+Q bound to `bindsym $mod+q kill`. This works well for all of my programs except one, Steam.

When I launch Steam, it creates a tray icon and throws up its windows. That's fine. But when I press Win+Q on the main Steam window, not only does the window close, but Steam exits entirely. In fact, if I have any Steam games running, those exit also.

Clicking the "X" button in Steam does close the window like I want: The window closes, but Steam stays in the tray. I would like Win+Q to have the same effect.

Is there a way to accomplish this?

**Comment by yusufaktepe** [comment]
This might work (install `wmctrl`):
```
bindsym $mod+q exec --no-startup-id wmctrl -c :ACTIVE:
```

**Reply by metov** [reply]
That seems to have the same problem. I tested by running:
```sh
sleep 5 && i3-msg 'exec --no-startup-id wmctrl -c :ACTIVE:'
```

Then switching to the Steam window and waiting. It closed the Steam window, but also Steam itself and the tray disappeared.
